modules/objc/examples/code_snippets/SampleCodeTest.m

```diff
@@ -934,6 +934,25 @@ - (void) dontTestReplicationPullFilter {
     // end::replication-pull-filter[]
 }
 
+//  Added 2/Feb/21 - Ian Bridge
+    - void dontTestCustomHeartbeat {
+    // tag::replication-set-heartbeat[]
+        id target =
+          [[CBLURLEndpoint alloc] initWithURL: [NSURL URLWithString: @"ws://foo.cbl.com/db"]];
+
+        CBLReplicatorConfiguration* config = [self configWithTarget: target
+                                                            type: kCBLReplicatorTypePush
+                                                      continuous: YES];
+
```

Comment:
can we use simple initialization for the ReplicatorConfiguration like 

```
CBLReplicatorConfiguration* config = [[CBLReplicatorConfiguration alloc] initWithDatabase: db target: target];
```

Reply:
@jayahariv  - see https://github.com/couchbase/docs-couchbase-lite/pull/401/commits/404291a807a98e6671c65d5000d66e147d06fa18